networks/known_networks.go

```diff
@@ -637,25 +638,52 @@ func MustGetSelectedNetworksFromEnv() []blockchain.EVMNetwork {
 			wsEnvVar := fmt.Sprintf("%s_URLS", networkKeys[i])
 			wsEnvVal := os.Getenv(wsEnvVar)
 			if wsEnvVal == "" {
-				panic(errors.Errorf("set %s env var", wsEnvVar))
+				// Get default value
+				defaultUrls, err := utils.GetEnv("EVM_URLS")
+				if err != nil {
+					panic(errors.Errorf("error getting %s EVM_URLS var", err))
+				}
+				if defaultUrls == "" {
+					panic(errors.Errorf("set %s or EVM_URLS env var", wsEnvVar))
+				}
+				wsUrls = strings.Split(defaultUrls, ",")
+			} else {
+				wsUrls = strings.Split(wsEnvVal, ",")
 			}
-			wsUrls = strings.Split(wsEnvVal, ",")
 
 			// Get network RPC HTTP URL from env var
 			httpEnvVar := fmt.Sprintf("%s_HTTP_URLS", networkKeys[i])
 			httpEnvVal := os.Getenv(httpEnvVar)
 			if httpEnvVal == "" {
-				panic(errors.Errorf("set %s env var", httpEnvVar))
+				// Get default value
+				defaultUrls, err := utils.GetEnv("EVM_HTTP_URLS")
+				if err != nil {
+					panic(errors.Errorf("error getting %s EVM_HTTP_URLS var", err))
+				}
+				if defaultUrls == "" {
+					panic(errors.Errorf("set %s or EVM_HTTP_URLS env var", httpEnvVar))
+				}
+				httpUrls = strings.Split(defaultUrls, ",")
+			} else {
+				httpUrls = strings.Split(httpEnvVal, ",")
 			}
-			httpUrls = strings.Split(httpEnvVal, ",")
 
 			// Get network wallet key from env var
 			walletKeysEnvVar := fmt.Sprintf("%s_KEYS", networkKeys[i])
 			walletKeysEnvVal := os.Getenv(walletKeysEnvVar)
 			if walletKeysEnvVal == "" {
-				panic(errors.Errorf("set %s env var", walletKeysEnvVar))
+				// Get default value
```

Comment:
See above

Reply:
@kalverra added in 61ddd40688da4b2e171753d84dd5ca2c29ef03e2